src/components/App/App.js

```diff
@@ -1,10 +1,14 @@
+import { throttle } from 'lodash';
 import flatMap from 'lodash/flatMap';
+import fetch from 'node-fetch';
```

Comment:
What is this needed for? You don't trust the browser support for `fetch`?

Reply:
Good point, this is unnecessary. I'll remove it.